Documentation/gettingstarted/hubble.rst

```diff
@@ -18,21 +18,22 @@ service map.
 
 .. note::
 
-   This guide assumes that Cilium has been correctly installed in your
-   Kubernetes cluster and that Hubble has been enabled. Please see
-   :ref:`k8s_quick_install` and :ref:`hubble_setup` for more information. If
-   unsure, run ``cilium status`` and validate that Cilium and Hubble are up and
-   running.
+   This guide assumes that Cilium and Hubble has been correctly installed in your
+   Kubernetes cluster. Please see :ref:`k8s_quick_install` and :ref:`hubble_setup`
+   for more information. If unsure, run ``cilium status`` and validate that Cilium
+   and Hubble are installed.
 
 Enable the Hubble UI
 ====================
 
-If you have not done so already, enable the Hubble UI by running the following command:
+Enable the Hubble with UI by running the following command:
```

Comment:
```suggestion
Enable the Hubble UI by running the following command:
```

Reply:
addressed in https://github.com/cilium/cilium/commit/de019848abd38aa1155c7fbd4711081505a25234